LESSON-06/katas/solution/personalLibrary.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+interface humorousBook{
+    public function containsHumor();
```

Comment:
You have greatly declared this interface, but you forgot, to create the containsHumour methods on the classes

Reply:
containsHumor -> containsHumour